CS4850Group5A/VirtuDoc
# Cache Maven dependencies between stack builds

# Summary
- Added mapped volume to dev Docker container that caches `/root/.m2`.
- Fixed issue where residual files in `target/` could cause a build failure.

Closes #10

# How to Test
1. Pull down branch.
2. Run `docker compose up --build` (or use the tools in your IDE). The stack build time may be somewhat longer than previously.
3. Press `Ctrl-C` to destroy the stack.
4. Run `docker compose up --build` again. This time the application should come up much faster, and not download 100 MB worth of dependencies on each run.

# Additional Comments
If subsequent builds are still very slow, or extremely slow compared to before, try moving the project to NVMe or SSD based storage (or a RAMdisk if you know how to set that up). If stack startup times are still slow, leave a blocking change request on this PR.
